### PR TITLE
Fix Spanish translation

### LINKED ISSUE #257: Improve Spanish translation
The Spanish translation is not very good, I can fix it later with a pull request.

### PR DESCRIPTION
**Pull request by jorgesumle**
It fixes #257

### CODE REVIEW

**Comment by agateau** [comment]
Looks promising, thanks!

**Comment by jorgesumle** [comment]
I'm done!

File: android/assets/po/es.po [hunk 1]
```diff
@@ -2,860 +2,867 @@
...
-msgstr  "Seleccionáis vuestro vehículos"
+msgid "Select Your Vehicles"
+msgstr "Seleccionad vuestros vehículos"
 
 #: android/assets/screens/multiplayer.gdxui:16
 #: android/assets/screens/multiplayer.gdxui:18
-msgid   "Ready!"
-msgstr  "¡ Listo !"
+msgid "Ready!"
+msgstr "¡Listo!\t"
```
Comment (agateau) [inline android/assets/po/es.po:395]: Are you sure about the `\t`?

File: android/assets/po/es.po [hunk 2]
```diff
@@ -2,860 +2,867 @@
...
 #: core/src/com/agateau/pixelwheels/RewardManagerSetup.java:104
 #: core/src/com/agateau/pixelwheels/RewardManagerSetup.java:128
-msgid   "Hit one vehicle with a missile"
-msgid_plural    "Hit %# vehicles with a missile"
-msgstr[0]       "Golpear uno vehículo con un misil"
-msgstr[1]       "Golpear %# vehículos con un misil"
+msgid "Hit one vehicle with a missile"
+msgid_plural "Hit %# vehicles with a missile"
+msgstr[0] "Golpea un vehículo con un misil"
+msgstr[1] "Golpea %# vehículos con un misil"
```
Comment (agateau) [inline android/assets/po/es.po:518]: Here the sentence does not use the infinitive form anymore. Should it be the case for the other rewards?

File: android/assets/po/es.po [hunk 3]
```diff
@@ -2,860 +2,867 @@
...
+msgstr[1] "Salir de la carretera %# veces"
 
 #: core/src/com/agateau/pixelwheels/RewardManagerSetup.java:120
-msgid   "Pick one bonus"
-msgid_plural    "Pick %# bonuses"
-msgstr[0]       "Coger uno bono"
-msgstr[1]       "Coger %# bonos"
+msgid "Pick one bonus"
+msgid_plural "Pick %# bonuses"
+msgstr[0] "Coger un bonificación\t"
```
Comment (agateau) [inline android/assets/po/es.po:530]: Another `\t` here.

**Review by agateau — CHANGES_REQUESTED** [review]
Looks good! Just a few minor questions.

File: android/assets/po/es.po (hunk 3 above)
Comment (jorgesumle) [reply android/assets/po/es.po:530]: It should be "Coger una bonificación".

File: android/assets/po/es.po (hunk 2 above)
Comment (jorgesumle) [reply android/assets/po/es.po:518]: Yes, "Leave road one time" should be "Sal de la carretera una vez".

**Comment by jorgesumle** [comment]
Fixed!

**Review by agateau — APPROVED** [review]
Sorry for the late reply. Looks good, merging it. Thanks!

**Pull request merged**